src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -78,6 +81,7 @@ public class ReplicationLoader : AbstractReplicationLoader<DocumentsContextPool,
         public ConflictSolver ConflictSolverConfig;
         private readonly CancellationToken _shutdownToken;
         private HubInfoForCleaner _hubInfoForCleaner;
+        private bool _replicationDisabledByMarker;
```

Comment:
do we want to make it public so we can override it JS admin console if needed?

Reply:
will it apply properly in runtime? what's required for it to come in effect then?